index.html

```diff
@@ -261,7 +338,7 @@ <h2 class="project-title">Multi-Post <br> Stories</h2>
                 </div>
 
                 <div class="project-button-link-container">
-                    <a href="https://github.com/DjibrilM/J-snippet/tree/main">
+                    <a href="#">
                         See project
                     </a>
                 </div>
```

Comment:
- [ ] Hello colleague, I could not highlight all the line of codes needed for this comment, but I would kindly advise you stick to the project requirement as much as possible, The goal of this milestone, is to have all your project cards that has been hardly coded in your HTML file be created dynamically in your JavaScript file using `array of an object`, Then all the line of code in your HTML file will be removed.

Below is the requirement 👇 

<img width="701" alt="image" src="https://github.com/DjibrilM/Portfolio/assets/78634841/eabd9edf-484a-4ab8-9924-deac038562cd">